### ISSUE TITLE
User accounts with saving cats (1000 pts)

### ISSUE THREAD

**Issue by wilsonmui** [ucsb-cs56-webapps/ucsb-cs56-catfinder#6]
Note this will require setting up authentication and a database. New templates will also likely need to be created. Users should be able to create a new account and login to it. Within their account, they can 'favorite' cats they like to save them. The users should be able to see their saved cats. If a saved cat is no longer available, it will no longer be on the list.

**Comment by johnnyzhang295** [comment]
I think you meant 100 points? @wilsonmui

**Comment by wilsonmui** [comment]
> I think you meant 100 points? @wilsonmui

No, I think implementing all this will be quite painful. It would require changing everything, so I think 1000 points is fair in my opinion. Getting all of this working should be difficult.

**Comment by johnnyzhang295** [comment]
Okay just making sure!
